recipe/install-libxgboost.sh

```diff
@@ -26,9 +26,9 @@ else
   EXEEXT=
 fi
 
+ls ${SRC_DIR}
 mkdir -p ${LIBDIR} ${INCDIR}/xgboost ${BINDIR} || true
 cp ${SRC_DIR}/xgboost${EXEEXT} ${BINDIR}/
```

Comment:
Is this executable still around or did it get removed in 2.1.0?

Reply:
Yes, let's remove it

Reply:
Cool that should fix CI: https://github.com/conda-forge/xgboost-feedstock/pull/167#issuecomment-2198017977